features/nanostack/mbed-mesh-api/mbed_lib.json

```diff
@@ -111,48 +111,59 @@
             "value": 255
         },
         "wisun-network-name": {
-            "help": "default network name for wisun network",
+            "help": "Network name for a wisun network. Maximum network name length can be 32 ASCII characters excluding terminating 0",
             "value": "\"Wi-SUN Network\""
         },
         "wisun-regulatory-domain": {
-            "help": "Regulator domain.",
+            "help": "Regulator domain value as specified in the Wi-SUN PHY Specification. Default value 3 is for EU region.",
+            "value_min": 0,
+            "value_max": 15,
             "value": "3"
         },
         "wisun-operating-class": {
-            "help": "Operating class. Use 255 to use Nanostack default",
-            "value": "255"
+            "help": "Operating class for the regulatory-domain as specified in the Wi-SUN PHY Specification. Default value 2 can be used in EU region.",
+            "value_min": 0,
+            "value_max": 42,
+            "value": "2"
         },
         "wisun-operating-mode": {
-            "help": "Operating mode. Use 255 to use Nanostack default",
-            "value": "255"
+            "help": "Operating mode as specified in the Wi-SUN PHY Specification. Default value 3 can be used in EU region and means 150ksymbol/s and modulation index 0.5.",
+            "value": "3"
         },
         "wisun-uc-channel-function": {
-            "help": "Unicast channel function.",
-            "value": "255"
+            "help": "Unicast channel function as specified in the Wi-SUN FAN specification.",
+            "accepted_values": [0, 1, 2, 3],
+            "value": 2
         },
         "wisun-bc-channel-function": {
-            "help": "Broadcast channel function.",
-            "value": "255"
+            "help": "Broadcast channel function as specified in the Wi-SUN FAN specification.",
+            "accepted_values": [0, 1, 2, 3],
+            "value": 2
         },
         "wisun-uc-fixed-channel": {
-            "help": "Default fixed channel",
-            "value": "0xffff"
+            "help": "Default 16-bit fixed channel for unicast. Used when channel hopping is not desired.",
+            "value_max": 65535‬,
+            "value": ‭65535‬
         },
         "wisun-bc-fixed-channel": {
-            "help": "Default fixed channel",
-            "value": "0xffff"
+            "help": "Default 16-bit fixed channel for multicast. Used when channel hopping is not desired.",
+            "value_max": 65535‬,
+            "value": 65535‬
         },
         "wisun-bc-interval": {
-            "help": "Broadcast interval. Duration between broadcast dwell intervals. Range: 0-16777216 milliseconds",
+            "help": "Broadcast interval. Duration between broadcast dwell intervals. 32-bit, keep as 0 to learn the value from network.",
```

Comment:
BC interval and dwell  is always learned by nodes and always set to value for border router. So when this value is set valid value should be used if this configuration is in future used by border router sw.

dwell:250, interval:1050 are the defaults if I remember correctly